src/app/packages/login/login/login.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../services/auth.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import { isNil as _isNil } from 'lodash';
+
+
+@Component({
+	selector: 'ansyn-login',
+	styleUrls: ['./login.component.less'],
+	templateUrl: 'login.component.html'
+})
+
+export class LoginComponent implements OnInit {
+
+	username: string;
+	password: string;
+	rememberMe: boolean;
+	returnUrl = this.authService.authrizedPath;
+	returnUrl$: Observable<string> = this.activatedRoute
+		.queryParams
```

Comment:
https://angular.io/guide/router#activatedroute-the-one-stop-shop-for-route-information